src/modules/cmdpal/Microsoft.CmdPal.UI/ExtViews/ListPage.xaml.cs

```diff
@@ -77,21 +76,23 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
                         // TODO: Handle failure case
                         System.Diagnostics.Debug.WriteLine(lvm.InitializeCommand.ExecutionTask.Exception);
 
-                        _ = _queue.EnqueueAsync(() =>
+                        // _ = _queue.EnqueueAsync(() =>
+                        _queue.TryEnqueue(new(() =>
```

Comment:
@michael-hawker I was hitting some weird conflict on `EnqueueAsync` here after adding `CommunityToolkit.WinUI.UI.Controls.Markdown`, any ideas why?

Reply:
![image](https://github.com/user-attachments/assets/c14a19ef-c521-4723-af25-93b39869941f)

Reply:
This is probably 7.x and 8.x not playing well together most likely as the namespaces/assemblies changed. Can we try the new labs Markdown version of the control? It's something that's getting picked up a bit more now again, so we hope we can make it stable eventually to replace the old one in a few months? 🤞 https://aka.ms/wct/wiki/previewpackages

https://dev.azure.com/dotnet/CommunityToolkit/_artifacts/feed/CommunityToolkit-Labs/NuGet/CommunityToolkit.Labs.WinUI.Controls.MarkdownTextBlock